src/atoms/Pagination/Pagination.js

```diff
@@ -7,11 +7,11 @@ import { prevIndex, nextIndex } from '../../utils/utils';
 
 const Pagination = ({
   elementClass,
-  elementsPerPage,
-  totalElements,
   currentPage,
+  totalPages,
   linkPrefix,
```

Comment:
I think calling this `slug` instead of `linkPrefix` would match our other stuff a little better